src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -526,45 +695,61 @@ private Vector2[] GetLeastIntersectingPath(PatchRegion start, PatchRegion end)
         var right = startRect.End.X > endRect.End.X ? startRect : endRect;
 
         // 3-segment line, Z shape
-        var middlePoint = new Vector2(left.End.X + right.Position.X, upper.End.Y + lower.Position.Y) / 2.0f;
+        for (int i = -3; i <= 3; ++i)
+        {
+            var middlePoint =
+                new Vector2(left.End.X + right.Position.X + i * 25, upper.End.Y + lower.Position.Y + i * 25) / 2.0f;
```

Comment:
The 25 here and the 3 few lines below seem like magic numbers I have no clue about, could you make those named const values in this class to clarify their reason why they are like that?